ftl/code/datums/sector.dm

```diff
@@ -32,6 +34,8 @@
 
   if(prob(station_prob))
     station = new /datum/station
+    
+  parallax_icon = pickweight(list("empty" = 3, "fog" = 1, "rocks" = 1)) //temp to prove it works

```

Comment:
rocks should later on be linked to whether or not we have asteroids in orbit or if we're in an event. this is fine for now though.